docs/release-notes/8.4.2-2.md

```diff
@@ -0,0 +1,33 @@
+# Percona XtraDB Cluster 8.4.2-2 (2024-12-11)
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
+
+## Release highlights
+
+Percona XtraDB Cluster is based on Percona Server for MySQL. Find a complete list of improvements and bug fixes in the [Percona Server for MySQL 8.4.2-2 (2024-11-04) release notes](https://docs.percona.com/percona-server/8.4/release-notes/8.4.2-2.html).
+
+## Bugs fixed
+
+[PXC-4495](https://perconadev.atlassian.net/browse/PXC-4495): PXC node creates new `gcache.page.X` but does not delete them.
+
+[PXC-4500](https://perconadev.atlassian.net/browse/PXC-4500): Cluster enters a broken state after failed node restart
+
+[PXC-4545](https://perconadev.atlassian.net/browse/PXC-4545): `percona_telemetry` is not able to fetch the `databases_count` with a mysql.session user
+
+[PXC-4546](https://perconadev.atlassian.net/browse/PXC-4546): The `databases_size` value in the json file created by `percona_telemetry` is zero.
+
+[PXC-4521](https://perconadev.atlassian.net/browse/PXC-4521): Percona Telemetry creates a dedicated, internal user limited to only collecting metrics. The user is created automatically when Percona Telemetry is enabled. If Percona Telemetry is disabled, the user is dropped during server startup.
+
+### Admin & maintenance task
+
+[PXC-4471](https://perconadev.atlassian.net/browse/PXC-4471): S3 bucket vulnerability found
```

Comment:
Please remove. This was just a cleanup. No real vulnerability was there, only unused directory. We removed the directory.